mixins/localize-mixin.js

```diff
@@ -97,8 +97,11 @@ export const LocalizeMixin = dedupeMixin(superclass => class LocalizeMixinClass
 
 	localize(key) {
 
+		const { language, value } = this.__resources?.[key] ?? {};
+		if (!value) return '';
+
 		let params = {};
-		if (arguments.length > 1 && typeof arguments[1] === 'object') {
+		if (arguments.length > 1 && arguments[1].constructor === Object) {
```

Comment:
This is matching `new String()` objects, which technically should work as a param name.